addon-resizer/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 FROM BASEIMAGE
-MAINTAINER Quintin Lee "qlee@google.com"
+LABEL maintainer="Quintin Lee <qlee@google.com>"
```

Comment:
Please just remove it.

Reply:
IIUC, we should remove this completely? @piosz